fp-bindgen/src/types/cargo_dependency.rs

```diff
@@ -32,15 +34,27 @@ impl CargoDependency {
                 git: Some(git),
                 branch: other.branch,
                 path: None,
+                workspace: None,
                 version: other.version.or(self.version),
                 features: self.features.union(&other.features).copied().collect(),
                 default_features: other.default_features.or(self.default_features),
             }
+        } else if let Some(workspace) = &other.workspace {
+            Self {
+                workspace: Some(*workspace),
+                git: None,
+                branch: None,
+                path: None,
+                version: None,
```

Comment:
I think we should explicitly set these, so if someone explicitly tries to override with `workspace: Some(false)` it would work as expected (otherwise I don't see why `workspace` is an `Option`...).

```suggestion
                git: other.git,
                branch: other.branch,
                path: other.path,
                version: other.version,
```